src/main/java/uk/gov/ida/verifyserviceprovider/services/EidasAssertionService.java

```diff
@@ -95,6 +105,12 @@ public LevelOfAssurance extractLevelOfAssuranceFrom(Assertion countryAssertion)
         }
     }
 
+    private Optional<AuthnContext> getAuthnContext(String uri) {
+        AuthnContext authnContext = authnContextFactory.mapFromEidasToLoA(uri);
+
+        return Optional.of(authnContext);
```

Comment:
This will through an Exception if the LOA is null

Reply:
Yep, it should not be null at this point...  moreover the hashing actually requires LOA (AuthnContext) and will throw an exception too anyway, so I'm not 100% sure why we're passing this around as optional 🤷‍♂️